inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,79 +1,399 @@
+website, among the tabs on top: → _Documentation_,
+https://inception-project.github.io/releases/0.10.0/docs/admin-guide.html[h]https://inception-project.github.io/releases/0.10.0/docs/admin-guide.html[ttps://inception-project.github.io/releases/0.10.0/docs/admin-guide.html]
 
-Alternatively, you can start it on the command line using
+Installing and starting INCEpTION
 
-[subs="+attributes"]
-----
-$ java -jar inception-app-standalone-{revnumber}.jar
-----
+* Installing Java
+** In order to make INCEpTION run, you will need to install Java (in
+case you do not have it yet). Here
+https://java.com/en/download/help/download_options.xml#linux you find
+instructions on how to install Java on Windows, Mac, Linux and Solaris.
 
-The splash screen and dialog will not appear in this case and you have to manually point your browser at `http://localhost:8080`.
+* Download and start INCEpTION
+** Now, download the .jar-file from our website
+https://inception-project.github.io/ . You will find the
+„Download“-Button top right. Wait a moment until it is fully downloaded.
+That is, until the name of the downlauded folder ends on „.jar“, not on
+„.jar.part“ anymore.
++
+_Note_: We recommend working on the latest version always. So check the
+website once in a while and download the new .jar-file when a later
+version is available.
+** Now, simply ***double-click ***on the downloaded .jar-file. After a
+moment, a splash screen will display while the application is
+initializing. Once the initialization is complete, a dialog appears.
+Here, you can open the application in your default browser or shut it
+down again.
+** If you are used to the command line, you may enter this command there
+instead:
++
+$ java -jar inception-app-standalone-0.10.0.jar
++
+In this case, no splash screen will start. Just enter
+http://localhost:8080/[_http://localhost:8080_] in you browser.
+** Use „admin“ as username and password to login, since a default user
+with these properties is created the first time you start the
+application.
 
-The first time you start the application, a default user with the name `admin` and the password
-`admin` is created. Use this username and password to log in to the application after opening it
-in your browser.
+* Download an example project for your first steps
+** Also on our website, clicking the tab „Example Projects“ on the very
+top leads you to the following page:
+https://inception-project.github.io/example-projects/ . To follow this
+Guide, please download the _Interactive Concept Linking_ project.
++
 
-== Your first annotation
+First Steps with INCEpTION
 
-After logging in, you will see the main menu. Click on *Projects* to go to the 
-*project management page* and there click on *create* to start a new project. Enter a name
-for your project, select the type *annotation project* and press *save*.
+* You may want to watch our tutorial video „Overview“ on how to create
```

Comment:
You need to explain core concepts of inception before letting the user annotate stuff
1. how is inception designed: project, multi-user, ...
1. what are projects
...

Reply:
Users genereally want to just start and _do_ things. If they have to read much theory first, they get bored, quit and look for an other tool so good tutorials get you to do what you want (here: annotate) really quick (for my taste, this has been too mich already).
If the first hunger is satisfied, people are open for background - which follows right away.

Reply:
I feel that if you mention stuff you need to explain it before. If you do not want to overwhelm users with information, make these sections optional, e.g. by using special formatted sections with info for more interested people or collapsable sections (accordions) that are hidden by default

Reply:
These are two ways of thinking. We will have both kinds of users so I want to meet both.

So I added that:
- we will only create annotations here and have no background explanantions
- if you are interestett in understanding the background first, we recommend reading "Anatomy of a Project" first
- There, I start with what a project is generally and then explain teh details ("anatomy") of a project
(This listing is concluded)

I think it should be clear then, right?
We might change the headline "Anatomy of a Project" since it coveres more now.

Reply:
Sounds ok

Reply:
This one should be relsolved since the didactical approach and aim have changed. 
Now: 
- theroartical explanation/orientation comes first, hands on (creating annotations) second
- both parts refer to the other one so those who first want to DO things can do so.